src/mpid/ch3/channels/nemesis/netmod/tcp/tcp_getip.c

```diff
@@ -280,14 +281,28 @@ int MPIDI_Get_IP_for_iface(const char *ifname, MPIDI_CH3I_nem_tcp_ifaddr_t *ifad
     if (found != NULL)
         *found = FALSE;
 
+	/*IPV6
+	if (ipv6_flag_set) 
+	fd = socket(AF_INET6, SOCK_DGRAM, 0);
+	else */
```

Comment:
If this code is not necessary, please simply remove it. Don't leave it as a comment.

Reply:
This comment is aimed to provide an indication that we need to switch between IPV6 socket and IPV4 socket here.